ksp_plugin/interface.cpp

```diff
@@ -195,6 +310,66 @@ void principia__DeletePlugin(Plugin const** const plugin) {
   LOG(INFO) << "Plugin destroyed";
 }
 
+void principia__DirectlyInsertMassiveCelestial(
+    Plugin* const plugin,
+    int const celestial_index,
+    int const* parent_index,
+    char const* gravitational_parameter,
+    char const* x,
+    char const* y,
+    char const* z,
+    char const* vx,
+    char const* vy,
+    char const* vz) {
+  CHECK_NOTNULL(plugin)->
+      DirectlyInsertCelestial(
+          celestial_index,
+          parent_index,
+          {Barycentric::origin +
+               Displacement<Barycentric>({ParseLength(x),
+                                          ParseLength(y),
+                                          ParseLength(z)}),
+               Velocity<Barycentric>({ParseSpeed(vx),
+                                      ParseSpeed(vy),
+                                      ParseSpeed(vz)})},
+          std::make_unique<MassiveBody>(
+              ParseGravitationalParameter(gravitational_parameter)));
+}
+
+void principia__DirectlyInsertOblateCelestial(
+    Plugin* const plugin,
+    int const celestial_index,
+    int const* parent_index,
+    char const* gravitational_parameter,
+    char const* axis_right_ascension,
+    char const* axis_declination,
+    char const* j2,
+    char const* reference_radius,
+    char const* x,
+    char const* y,
+    char const* z,
+    char const* vx,
+    char const* vy,
+    char const* vz) {
```

Comment:
Here I would like to compute the dimensionful j2 and create a non-oblate body if it's smaller than some threshold.  It's fine to have the most correct data in the config file, but oblate bodies are super-expensive and the j2 of, say, Venus or Mercury probably has no effect on anything, even close to these bodies.  It may well be that this explains the slowness when warping.  So from a practical perspective a cutoff would make a lot of sense.